client/coral-admin/src/graphql/mutations/index.js

```diff
@@ -54,7 +54,20 @@ export const setCommentStatus = graphql(SET_COMMENT_STATUS, {
         },
         updateQueries: {
           ModQueue: (oldData) => {
-            const comment = oldData.all.find(c => c.id === commentId);
+            let comment = oldData.all.find(c => c.id === commentId);
+
+            if (!comment) {
+              comment = oldData.premod.find(c => c.id === commentId);
+            }
+
+            if (!comment) {
+              comment = oldData.flagged.find(c => c.id === commentId);
+            }
+
+            if (!comment) {
+              comment = oldData.rejected.find(c => c.id === commentId);
+            }
+
```

Comment:
I don't know if that's more pretty. it's definitely more "es6", but much harder to parse.